src/main/kotlin/org/sert2521/infiniterecharge2020/drivetrain/DrivetrainCommands.kt

```diff
@@ -84,3 +109,25 @@ suspend fun <T : MetricUnit<Linear>> driveCurve(
         }
     }
 }
+
+suspend fun runPath(
+    drivetrain: Drivetrain,
+    trajectory: Trajectory,
+    getPose: () -> Pose2d = { drivetrain.pose },
+    follower: RamseteController = RamseteController(),
+    pathKinematics: DifferentialDriveKinematics = kinematics,
+    outputMetersPerSecond: (left: Double, right: Double) -> Unit = { l, r -> drivetrain.setTargetSpeed(l.mps, r.mps) }
+) {
+    timer(20, 0, (trajectory.totalTimeSeconds * 1000).toLong().also { println("Path time: $it") }) {
```

Comment:
Why are you multiplying seconds by 1000?

Reply:
I think that the timer function takes in milliseconds as an argument.

Reply:
Do we want to update the name to reflect that? `totalTimeMilli`?